src/test/java/org/broadinstitute/barclay/argparser/CommandLinePluginUnitTest.java

```diff
@@ -212,7 +212,8 @@ public void testBasicPlugin(final String[] args, final int expectedInstanceCount
         PlugInTestObject plugInTest = new PlugInTestObject();
         final CommandLineArgumentParser clp = new CommandLineArgumentParser(
                 plugInTest,
-                Collections.singletonList(new TestPluginDescriptor(Collections.singletonList(new TestDefaultPlugin()))));
+                Collections.singletonList(new TestPluginDescriptor(Collections.singletonList(new TestDefaultPlugin()))),
+                Collections.<CommandLineParserOptions>emptySet());
```

Comment:
You specified the types of this collection explicitly in a number of places.  I don't think it's necessary, but if you think that it adds clarity then it's fine.